File: Study Note/Algorithm Problems/SWEA/D3/5431 민석이의 과제 체크하기 (pass)/SWEA_5431.py
import sys
sys.stdin = open("input.txt", "r")

T = int(input())
for test_case in range(1, T + 1):
    # ///////////////////////////////////////////////////////////////////////////////////
    '''

        이 부분에 여러분의 알고리즘 구현이 들어갑니다.

    '''
    # ///////////////////////////////////////////////////////////////////////////////////
    N, K = map(int, input().split())
    count = [0] * N
    submit = list(map(int, input().split()))
    for s in submit:
        count[s-1] += 1
    print('#{}'.format(test_case), end='')
    for i in range(N):
        if not count[i]:
            print(' {}'.format(i + 1), end='')
    print()

File: Study Note/Algorithm Problems/SWEA/D3/5431 민석이의 과제 체크하기 (pass)/SWEA_5431.c
#include <stdio.h>
int main(void)
{
	int test_case;
	int T, N, K, i, stu;

	scanf("%d", &T);
	for (test_case = 1; test_case <= T; ++test_case)
	{
		scanf("%d %d", &N, &K);
      int count[N];
      for (i = 0; i < N; i++) 
        count[i] = 0;
      for (i = 0; i < K; i++)
      {
        scanf("%d", &stu);
        count[--stu]++;
      }
      printf("#%d", test_case);
      for (i = 0; i < N; i++)
        if (!count[i])
          printf(" %d", i + 1);
      printf("\n");
	}
	return 0;
}